test/TestProjects/ModelShapes/Generated/Models/InputModel.cs

```diff
@@ -24,7 +24,7 @@ public partial class InputModel
         /// <param name="requiredNullableInt"></param>
         /// <param name="requiredNullableStringList"></param>
         /// <param name="requiredNullableIntList"></param>
-        /// <exception cref="ArgumentNullException"> <paramref name="requiredString"/>, <paramref name="requiredStringList"/>, or <paramref name="requiredIntList"/> is null. </exception>
+        /// <exception cref="ArgumentNullException"> <paramref name="requiredString"/>, <paramref name="requiredStringList"/> or <paramref name="requiredIntList"/> is null. </exception>
```

Comment:
Is validating model properties the right thing to do?  I think these are server parameters.

Reply:
Autorest client again - pls disregard.